packages/jest-jasmine2/src/index.js

```diff
@@ -161,9 +161,10 @@ const addSnapshotData = (results, snapshotState) => {
   });
 
   const uncheckedCount = snapshotState.getUncheckedCount();
-  const uncheckedKeys = snapshotState.getUncheckedKeys();
+  let uncheckedKeys;
 
   if (uncheckedCount) {
+    uncheckedKeys = snapshotState.getUncheckedKeys();
```

Comment:
why was this changed? This seems unrelated.

Reply:
i got runtime errors when developing this feature when `uncheckedCount` is falsy. do you think that this change is not required?

Reply:
I think this should be changed in a separate commit, with a test that is fixed by your change. Please revert it from this PR and send one for this issues specifically. Thank you.